### PR TITLE
Add org.gimp.GIMP.Plugin.BIMP

### PR DESCRIPTION
**Pull request by hfiguiere**
This is  the second GIMP plugin

It is waiting on https://github.com/flathub/org.gimp.GIMP/pull/75 to be built.

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.gimp.GIMP.Plugin.BIMP

**Comment by bilelmoussaoui** [comment]
bot, build org.gimp.GIMP.Plugin.BIMP

File: org.gimp.GIMP.Plugin.BIMP.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.gimp.GIMP.Plugin.BIMP</id>
+  <extends>org.gimp.GIMP.desktop</extends>
```
Comment (bilelmoussaoui) [inline org.gimp.GIMP.Plugin.BIMP.metainfo.xml:4]: The correct id is `org.gimp.GIMP` https://gitlab.gnome.org/GNOME/gimp/-/blob/master/desktop/org.gimp.GIMP.appdata.xml.in.in#L4
```suggestion
  <extends>org.gimp.GIMP</extends>
```

Comment (bilelmoussaoui) [reply org.gimp.GIMP.Plugin.BIMP.metainfo.xml:4]: Same comment on other extensions, otherwise lgtm

Comment (hfiguiere) [reply org.gimp.GIMP.Plugin.BIMP.metainfo.xml:4]: to be fair I forgot where I picked up the .desktop but it's everywhere :-/ Will fix.

well actually this is where I picked it up:

https://github.com/flathub/io.github.Hexchat.Plugin.OTR/blob/7ec0c67ae98d186982db94b29cb9b2f58b866a6a/io.github.Hexchat.Plugin.OTR.metainfo.xml#L4

Comment (hfiguiere) [reply org.gimp.GIMP.Plugin.BIMP.metainfo.xml]: done

**Comment by nedrichards** [comment]
bot, build org.gimp.GIMP.Plugin.BIMP

**Pull request closed without merging**